### PR TITLE
Ewl 580 header breadcrumb

### PR DESCRIPTION
**Pull request by arturkin**
https://eurosportdigital.atlassian.net/browse/EWL-580

Test link: https://deploy-preview-136--es-web-toolkit.netlify.com/?selectedKind=Modules%7CHeader&selectedStory=configurable&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs&background=%23060825

### CODE REVIEW

File: src/modules/Header/index.js [hunk 1]
```diff
@@ -44,14 +47,87 @@ const StyledWrapper = styled.header`
...
 
     ${breakpoints.medium(css`
       margin: 0 25px 0 17px;
     `)}
   }
 `;
 
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: ${coreLightMinus1};
```
Comment (davidgarry) [inline src/modules/Header/index.js]: The color of the last item is ok, but the previous items must not be white (same behavior in production currently : https://www.eurosport.fr/tennis/roland-garros/)
image

File: src/modules/Header/index.js [hunk 2]
```diff
@@ -100,15 +176,18 @@ class Header extends Component {
...
-    const { homePageUrl, cta } = this.props;
+    const { homePageUrl, cta, breadcrumbs } = this.props;
 
     return (
       <StyledWrapper {...this.props} data-test="header">
         {this.getBurgerMenu()}
         {this.getBurgerIcon()}
         <Link href={homePageUrl} data-test="header-logo">
-          <Logo />
+          <HideOnMobile>
```
Comment (davidgarry) [inline src/modules/Header/index.js]: We must display the logo on mobile if there is no breadcrumb

File: src/modules/Header/index.js [hunk 3]
```diff
@@ -100,15 +176,18 @@ class Header extends Component {
...
+            <Logo />
+          </HideOnMobile>
         </Link>
+        {breadcrumbs && <Breadcrumbs items={breadcrumbs} />}
```
Comment (davidgarry) [inline src/modules/Header/index.js]: You must test that there is at least one item inside to generate the Breadcrumb component.

Comment (arturkin) [reply src/modules/Header/index.js]: Oh yes, completely forgot that, thanks!

File: src/modules/Header/index.js (hunk 2 above)
Comment (arturkin) [reply src/modules/Header/index.js]: Oh yes, completely forgot that, thanks!

File: src/modules/Header/index.js (hunk 1 above)
Comment (arturkin) [reply src/modules/Header/index.js]: done!

File: src/modules/Header/index.js (hunk 2 above)
Comment (arturkin) [reply src/modules/Header/index.js]: Done

File: src/modules/Header/index.js (hunk 3 above)
Comment (arturkin) [reply src/modules/Header/index.js]: done 🤭

**Pull request merged**